config.js

```diff
@@ -4,15 +4,14 @@ dotenv.config();
 const config = {
   AUTHPORT: process.env.PORT || "3002",
   ORIGIN: process.env.ORIGIN || "http://localhost:3002",
-  APPLICATIONORIGIN:
-    process.env.APPLICATION_ORIGIN || "http://localhost:3002/v1/web/home",
-  ADMINORIGIN:
-    process.env.ADMIN_ORIGIN || "http://localhost:3002/v1/admin/dashboard",
+  APPLICATIONORIGIN: process.env.APPLICATION_ORIGIN || "http://localhost:3002/v1/web/home",
+  REGISTRATIONORIGIN: process.env.REGISTRATION_ORIGIN || APPLICATIONORIGIN,
```

Comment:
Doesn't seem to work 
`ReferenceError: APPLICATIONORIGIN is not defined`

Reply:
check https://github.com/authcompanion/authcompanion2/pull/19/commits/af0d99c87d270bcbef9076ced55792507647c43a if that works.